### PR TITLE
add WormTabStrip

### PR DESCRIPTION
**Pull request by EzimetYusup**
## Project URL
https://github.com/EzimetYusup/WormTabStrip

## Category
UI->Tab Bar

## Description
Worm Tab Strip is inspired by android [SmartTabStrip](https://github.com/ogaclejapan/SmartTabLayout), android  view pager like library for iOS written in swift.

Basically it was build up by two scroll view, one at the top for holding all the tabs, one for content view for each tab.
frame based, not auto layout constraint based.
 
## Why it should be included to `awesome-ios` (optional)
because it is awesome
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 4 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 3 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
753 | [301](https://httpstatuses.com/301) | https://github.com/knly/Evergreen redirects to<br>https://github.com/nilsleiffischer/Evergreen
1416 | [301](https://httpstatuses.com/301) | https://github.com/skylab-inc/Edge redirects to<br>https://github.com/skylab-inc/Lightning
1796 | [301](https://httpstatuses.com/301) | https://github.com/storehouse/Advance redirects to<br>https://github.com/timdonnelly/Advance


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -2266,7 +2266,7 @@ Most of these are paid services, some have free tiers.
 * [SwipeableTabBarController](https://github.com/marcosgriselli/SwipeableTabBarController) - UITabBarController with swipe interaction between its tabs. 🔶
 * [SMSwipeableTabView](https://github.com/smahajan28/SMSwipeableTabView) - Swipeable Views with Tabs (Like Android SwipeView With Tabs Layout) 🔶
 * [Tabman](https://github.com/uias/Tabman) - A powerful paging view controller with indicator bar for iOS. 🔶
-
+* [WormTabStrip](https://github.com/EzimetYusup/WormTabStrip) Beatiful ViewPager For iOS written in Swift 🔶 (inspired by Android [SmartTabLayout](https://github.com/ogaclejapan/SmartTabLayout))
 #### Table View / Collection View
```
Comment (dkhamsing) [inline README.md]: We need a new line here pls

**Comment by EzimetYusup** [comment]
@dkhamsing  done, Thanks!

**Comment by dkhamsing** [comment]
Thank you

**Pull request merged**